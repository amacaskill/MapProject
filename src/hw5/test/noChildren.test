CreateGraph graph1

AddNode graph1 n1

ListChildren graph1 n1 