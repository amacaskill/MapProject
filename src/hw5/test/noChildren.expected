created graph graph1

added node n1 to graph1

the children of n1 in graph1 are: